src/apps/experimental/routes/user/menu/index.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { FunctionComponent, useCallback } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import List from '@mui/material/List';
+import ListItemMui from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import PersonIcon from '@mui/icons-material/Person';
+import TvIcon from '@mui/icons-material/Tv';
+import HomeIcon from '@mui/icons-material/Home';
+import PlayCircleFilledIcon from '@mui/icons-material/PlayCircleFilled';
+import ClosedCaptionIcon from '@mui/icons-material/ClosedCaption';
+import DevicesOtherIcon from '@mui/icons-material/DevicesOther';
+import KeyboardIcon from '@mui/icons-material/Keyboard';
+import Typography from '@mui/material/Typography';
+
+import { appHost } from 'components/apphost';
+import { useApi } from 'hooks/useApi';
+import globalize from 'scripts/globalize';
+import layoutManager from 'components/layoutManager';
+import Page from 'components/Page';
+import { usePreferencesMenuUser } from './hooks/usePreferencesMenuUser';
+
+const UserPreferencesMenu: FunctionComponent = () => {
+    // This page can also be used by admins to change user preferences from the user edit page
+    const [urlParams] = useSearchParams();
+    const { user: currentUser } = useApi();
+    const userId = urlParams.get('userId') || currentUser?.Id;
+    const { user } = usePreferencesMenuUser({ userId });
+
+    // Hide the actions if user preferences are being edited for a different user
+    const isForDifferentUser = urlParams.get('userId') && urlParams.get('userId') !== currentUser?.Id;
+
+    const onClientSettingsClick = useCallback(() => {
+        window.NativeShell?.openClientSettings();
+    }, []);
+
+    return (
+        <Page
+            id='myPreferencesMenuPage'
+            className='libraryPage userPreferencesPage noSecondaryNavPage'
+            title={globalize.translate('Settings')}
+        >
+            <div className='padded-left padded-right padded-bottom-page padded-top'>
+                <div className='readOnlyContent' style={{ margin: '0 auto' }}>
```

Comment:
This `readOnlyContent` doesn't seem to actually have anything to do with read only content, and just sets some responsive max widths. I wonder if we could just replace this `div` and class name with a [Box](https://mui.com/system/react-box/) component.

Reply:
I don't think that the Box component would help here. By default it's just div that doesn't have any breakpoints, so we would still need to define max-width somehow, rather via a className, since the `readOnlyContent` class is used in several components.